Content.Server/Chemistry/ReagentEffects/HealthChange.cs

```diff
@@ -25,15 +25,15 @@ public sealed class HealthChange : ReagentEffect
         /// </summary>
         [JsonPropertyName("scaleByQuantity")]
         [DataField("scaleByQuantity")]
-        public bool ScaleByQuantity = false;
+        public bool ScaleByQuantity = true; // Parkstation-fixReagentSuperPotency
 
         [DataField("ignoreResistances")]
         [JsonPropertyName("ignoreResistances")]
         public bool IgnoreResistances = true;
 
         public override void Effect(ReagentEffectArgs args)
         {
-            var scale = ScaleByQuantity ? args.Quantity : FixedPoint2.New(1);
+            var scale = ScaleByQuantity ? args.Quantity * 2 : FixedPoint2.New(1);  // Parkstation-fixReagentSuperPotency
```

Comment:
Could you explain what this is doing? I assume the actual damage is halved, and so we're doubling how much it deals as it ingests? 
Same amount overall, but digested slower.